clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -87,12 +91,51 @@ export interface UpdateEnabledIds {
   modelType: keyof EnabledIds;
 }
 
+type Category =
+  | "purpose_consent"
+  | "purpose_legitimate_interests"
+  | "special_feature"
+  | "vendor_consent"
+  | "vendor_legitimate_interests"
+  | "system_consent"
+  | "system_legitimate_interests";
+type NoticeSubMap = {
```

Comment:
these aren't technically Notices (we typically don't use notices for TCF). maybe we can change this to be `TcfEntitiesMap`?